inst/extdata/abm.Rmd

```diff
@@ -167,7 +167,30 @@ Statistics regarding citations and co-publishing are based on the subset of publ
 Row
 ----------------------
 
-### Disclaimer
+### Publication data and attributions
+
+```{r, echo=FALSE, out.height="60px"}
+
+embed_data <- function(path)
+  paste0("data:", mime::guess_type(path), ";base64,", base64enc::base64encode(path))
+
+embed_file_link <- function(path, 
+  href = embed_data(path), 
+  name = basename(path),
+  text = paste("Download", name), ...) {
+  htmltools::a(text, href = href, download = name, ...)
+}
+
+icon_download <- htmltools::tag("i", list(class = "fa fa-download"))
+icon_download <- htmltools::attachDependencies(icon_download, 
+  flexdashboard:::html_dependencies_fonts(TRUE, FALSE))
+
+embed_file_link("~/Downloads/PublList_KTH.xlsx", 
```

Comment:
embed_file_link("~/Downloads/PublList_KTH.xlsx", ...) gives
> Error in file(what, "rb") : cannot open the connection
> Calls: <Anonymous> ... dots_list -> embed_data -> paste0 -> <Anonymous> -> file
> In addition: Warning message:
> In file(what, "rb") :
>   cannot open file 'C:/Users/Agne/Documents/Downloads/PublList_KTH.xlsx': No such file or directory
> Execution halted

Reply:
Can this data be fetched from the db, if so from where? Otherwise, please download this file from and try again, link:

https://intra.kth.se/bibliometri/public/link/university/KTH/file/KTH.download

Reply:
Can this data be fetched from the db, if so from where? Otherwise, please download this file and put it in that location and try again, link:

https://intra.kth.se/bibliometri/public/link/university/KTH/file/KTH.download

Reply:
If the publication list data could be returned through a function, say, `abm_publications(unit_code == "KTH")`, then the .Rmd code above could change to `embed_file_link(abm_publications(unit_code == "KTH"))` instead.

Reply:
The .html for the generated flexdashboard goes up to 13M for me when embedding the 30000+ rows of publication list data.

Reply:
Aha, problem was that I had no ~/Documents/Downloads/ folder (I have Downloads directly under "$HOME", I think this is the standard location in Win).

Anyway, function for fetching from database coming up shortly.